pkg/model/model.go

```diff
@@ -31,14 +31,11 @@ import (
 )
 
 const (
-	idlePower = true  // idlePower is used to define if the function will update idle power
-	absPower  = false // absPower is used to define if the function will NOT update idle power, but instead an absolute power
-	gauge     = true  // gauge is used to define if the function will update a gauge value
-	counter   = false // gauge is used to define if the function will NOT update a gauge value, but instead a counter value
-)
-
-var (
-	EstimatorSidecarSocket = "/tmp/estimator.sock"
+	idlePower              = true  // idlePower is used to define if the function will update idle power
+	absPower               = false // absPower is used to define if the function will NOT update idle power, but instead an absolute power
+	gauge                  = true  // gauge is used to define if the function will update a gauge value
+	counter                = false // gauge is used to define if the function will NOT update a gauge value, but instead a counter value
```

Comment:
```suggestion
	counter                = false // counter is used to define if the function will update a counter value
```

Reply:
why changed these from `const` to `var` ?

Reply:
I changed `estimatorSidecarSocket = "/tmp/estimator.sock` from var to const because it's value is not updated anywhere that I could see. 
The variables above were always const. I didn't change anything only the alignment of the `=` to match the new const below

Reply:
aah ! my mistake. sorry